stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -278,6 +278,11 @@ spec:
               containerPort: {{ .Values.metrics.port  }}
           resources:
 {{ toYaml .Values.metrics.resources | indent 12 }}
+          {{- if .Values.securityContext.enabled }}
```

Comment:
This should be `.Values.metrics.securityContext.enabled` and add variable to values.yaml and README.